overlayer/textractoverlayer/image_tools.py

```diff
@@ -22,16 +22,18 @@ def get_size_from_filestream(fs, ext) -> DocumentDimensions:
         return DocumentDimensions(doc_width=img.width, doc_height=img.height)
     else:
         input1 = PdfFileReader(fs)
-        pdf = input1.getPage(0).mediaBox
-        return DocumentDimensions(doc_width=pdf.width, doc_height=pdf.height)
+        pdf_page = input1.getPage(0).mediaBox
+        return DocumentDimensions(
+            doc_width=float(pdf_page.getWidth()), doc_height=float(pdf_page.getHeight())
```

Comment:
Can you convert to int? DocumentDimensions expect int atm

        return DocumentDimensions(doc_width=int(pdf_page.getWidth()), doc_height=int(pdf_page.getHeight()))